src/index.js

```diff
@@ -0,0 +1,38 @@
+import './style.css';
+import render from './modules/render.js';
+import add from './modules/add.js';
+import completed from './modules/completed.js';
+
+// RENDER WHEN THE PAGE LOADS
+render();
+
+// ADD A NEW TASK WHEN CLICK ON ENTER BUTTON
+const addButton = document.querySelector('.add-button');
+addButton.addEventListener('click', () => {
+  add();
+  render();
+});
+
+// ADD A NEW TASK WHEN TYPE ENTER KEY
+const addTask = document.querySelector('.add-task');
+addTask.addEventListener('keydown', (event) => {
+  if (event.keyCode === 13) {
+    add();
```

Comment:
* As stated in the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Create%20an%20index.js,values%20for%20each%20task.) Your index.js file should have the following below, Kindly fix this 

> Create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object should contain three keys:
> description [string].
> completed [bool].
> index: [number].
> Write a function to iterate over the tasks array and populate an HTML list item element for each task.
> On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

Reply:
dear @Davidosky007 I've setted the array in the add.js file, should I remove the js modules and just use the index.js file? 
![image](https://user-images.githubusercontent.com/116839451/228733226-83262239-4ad8-4a1d-98ce-d7e92d4ebbc5.png)

Reply:
According to the requirements you should enter manually tasks in the array and render it according to their index